### ISSUE TITLE
Please keep US States in the time series folder (either in the global or a separate file)

### ISSUE THREAD

**Issue by lmirny** [CSSEGISandData/COVID-19#1352]
Please keep US States in the time series folder (either in the global or a separate file)

**Comment by dhmacq** [comment]
I think that's exactly what they said they would do in #1250.

**Comment by coder999** [comment]
Agree!  State-level data is missing from the time series data and is only in the daily reports now.
